package.json

```diff
@@ -44,36 +44,31 @@
     "babel-cli": "6.18.0",
     "babel-core": "6.20.0",
     "babel-eslint": "7.1.1",
+    "babel-jest": "^18.0.0",
```

Comment:
I agree with Cory - some packages dont follow versioning properly and introduce breaking changes when they shouldnt...it is best to be specific and control the version you upgrade to next